assets/dns/daemonset.yaml

```diff
@@ -0,0 +1,62 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: dns
+  namespace: openshift-cluster-dns
+  labels:
+    openshift-app: dns
+spec:
+  selector:
+    matchLabels:
+      openshift-app: dns
+  template:
+    metadata:
+      labels:
+        openshift-app: dns
+    spec:
+      serviceAccountName: dns
+      containers:
+      - name: dns
+        image: "docker.io/openshift/origin-coredns:v3.11.0"
+        imagePullPolicy: IfNotPresent
+        command: [ "coredns" ]
+        args: [ "-conf", "/etc/coredns/Corefile" ]
+        volumeMounts:
+        - name: config-volume
+          mountPath: /etc/coredns
+          readOnly: true
+        ports:
+        - containerPort: 53
```

Comment:
Technically there's no reason for the container itself to bind to a privileged port; the service can map a privileged port on its own IP to a named unprivileged container port.